doc/team_practices.md

```diff
@@ -0,0 +1,37 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
```

Comment:
Not sure that this is necessary for refactoring...anyone else have strong feelings?